docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -75,11 +76,9 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
  C# 4.0 [Ecma-334.pdf](https://www.ecma-international.org/publications/files/ECMA-ST/Ecma-334.pdf) Standard ECMA-334 4th Edition    
```

Comment:
This link now goes to C# 5.0. I'm looking for the C# 4.0 spec download. We may merge this without fixing that, because I can't find a download link for 4.0 yet.

Reply:
@BillWagner what about this one:
https://www.ecma-international.org/publications/files/ECMA-ST-ARCH/Ecma-334%204th%20edition%20June%202006.pdf

Reply:
No, that link is not correct, June 2006 is too early for C# 4.